src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/DirectoryControl.cs

```diff
@@ -836,8 +1004,7 @@ public VlvRequestControl(int beforeCount, int afterCount, string target) : this(
             AfterCount = afterCount;
             if (target != null)
             {
-                UTF8Encoding encoder = new UTF8Encoding();
-                _target = encoder.GetBytes(target);
+                _target = s_utf8Encoding.GetBytes(target);
             }
```

Comment:
This change makes invalid inputs (such as `"\uDD74\uD800"`) throw an `EncoderFallbackException`, which it used to not do.

I'm always a fan of rejecting nonsense inputs; but it's an observable change so it needs to have a test, and documented as a breaking change.

Reply:
@edwardneal is there a test for the exception semantics above?

Reply:
I've added `VlvRequestControlTests.Ctor_InvalidUtf8Target_ThrowsEncoderFallbackException`.